tests/benchmarks/test_model_schema_generation.py

```diff
@@ -262,3 +267,54 @@ def validate_model_wrap(cls, values: Any, handler: ModelWrapValidatorHandler[Sel
                 return handler(values)
 
     benchmark(schema_gen)
+
+
+@pytest.mark.parametrize('mode', ['plain', 'wrap'])
+@pytest.mark.benchmark(group='model_schema_generation')
+def test_custom_field_serializer(benchmark, mode) -> None:
+    def schema_gen() -> None:
+        class ModelWithFieldSerializer(BaseModel):
+            field1: int
+
+            @field_serializer('field1', mode=mode)
+            def serialize_field(cls, v: int) -> str:
+                return str(v)
+
+    benchmark(schema_gen)
```

Comment:
I think we need to separate the tests for `plain` and `wrap` serializers - they have different signatures (ex - see the different signatures for the corresponding validators). This looks like a great test for the `plain` serializer, though.

Reply:
(Just like you did for the annotated ones).

Reply:
You are right! I forgot about the signature difference. I believe it should be correct now.